README.md

```diff
@@ -66,4 +66,16 @@ The end-to-end tests are defined in the `e2e.bats` file and can be run using:
 
 ```console
 make e2e
-```
\ No newline at end of file
+```
+
+## Required Development Tools
+- [TypeScript](https://www.typescriptlang.org)
+- [Webpack](https://webpack.js.org)
+- [Prettier](https://prettier.io)
+- [Node.js](https://nodejs.org)
+- Make 
+- [bats (Bash Automated Testing System)](https://github.com/bats-core/bats-core)
+- [kwctl](https://github.com/kubewarden/kwctl/releases)
+- [javy](https://github.com/bytecodealliance/javy)
+- [kubewarden-policy-sdk](https://github.com/kubewarden/policy-sdk-js)
+- [kubernetes-types](https://github.com/silverlyra/kubernetes-types)
```

Comment:
I think these ones can be skipped. This list should contain only the tools (as in binaries) that have to be installed.